paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/AddPaymentMethod.kt

```diff
@@ -99,7 +97,6 @@ internal fun AddPaymentMethod(
             val stripeIntent = paymentMethodMetadata?.stripeIntent
 
             PaymentElement(
-                formViewModelSubComponentBuilderProvider = sheetViewModel.formViewModelSubComponentBuilderProvider,
```

Comment:
With the Dagger provider removed, would this unblock adding tests for `PaymentElement`? Are there additional changes that are planned for the `PaymentElement` component?

Reply:
Interesting. I hadn't made that connection yet. But I think you're right! This unlocks quite a bit of MUCH more straightforward testing in `PaymentElement` and `PaymentMethodForm`!